### ISSUE TITLE
Tensorflow lite int8 quantisation for depth_to_space() op

### ISSUE THREAD

**Issue by Cy-r0** [tensorflow/tensorflow#45213]
**System information**
- TensorFlow version (you are using): 2.3.1
- Are you willing to contribute it (Yes/No): No(t now)

**Describe the feature and the current behavior/state.**
The operation depth_to_space() currently seems to not be supported for quantisation to 8-bit in tensorflow lite.

**Who will benefit with this feature?**
depth_to_space() is frequently used in super-resolution models and sometimes in models that do upsampling. 
These models are usually more computationally expensive than traditional classifiers because of their massive intermediate feature maps; hence, they would greatly benefit from int8 quantisation when deployed on mobile hardware, but that's not possible without int8 support for depth_to_space().
Is this in the works? Has it been planned for a release?

Best

**Comment by Cy-r0** [comment]
Hello?

**Comment by karimnosseir** [comment]
Should be supported now. You can try with later nightly builds.

Thanks

**Comment by Cy-r0** [comment]
Hey, thank you for your help.

I installed the latest nightly version of tensorflow (2.5.0.dev20201210), converted a TensorFlow model using the depth_to_space() op to TFLite with int8 quantisation (no float fallback), and tried running it on an android device. 

When using NNAPI, the following error appears:
```
I/tflite: Created TensorFlow Lite delegate for NNAPI.
E/org.tensorflow.benchmarking.MainActivity: Error initialising tflite interpreter
    java.lang.IllegalArgumentException: Internal error: Cannot create interpreter: Didn't find op for builtin opcode 'DEPTH_TO_SPACE' version '2'
```
Am I supposed to compile tensorflow lite myself, or is there something else I'm missing? 

The build.gradle file in the app I wrote imports tensorflow lite as below:
```
dependencies {
    implementation 'androidx.appcompat:appcompat:1.1.0'
    // Build off of nightly TensorFlow Lite
    implementation('org.tensorflow:tensorflow-lite:0.0.0-nightly') { changing = true }
    implementation('org.tensorflow:tensorflow-lite-gpu:0.0.0-nightly') { changing = true }
    implementation('org.tensorflow:tensorflow-lite-support:0.0.0-nightly') { changing = true }
}
```

I'd be grateful if you could point me in the right direction, and let me know if you need any more information.

**Comment by karimnosseir** [comment]
That means the TFLite runtime you're using is not the latest.
There is an issue with Jcenter not up to date which we are looking into it (might be the reason).
Can you access this file and use it
 https://bintray.com/google/tensorflow/download_file?file_path=org%2Ftensorflow%2Ftensorflow-lite%2F0.0.0-nightly%2Ftensorflow-lite-0.0.0-nightly.aar

**Comment by Cy-r0** [comment]
Thanks for the file.

In addition to that, I also had to fetch the .aar archive for tensorflow-lite-support from Jcenter manually, otherwise an error popped up, which I'm chalking up to some confict between the local module for tensorflow-lite and the remote module for tensorflow-lite-support: 
```
More than one file was found with OS independent path 'lib/arm64-v8a/libtensorflowlite_jni.so'
```
With both modules installed locally, I was able to build and run the app.

Something interesting I noticed, the int8 quantised version of depth_to_space() doesn't seem to provide any speed advantage over the float version (in a model quantised with float fallback). Is there any way to check in Android what's happening inside the model, e.g. whether it uses only int8 operations?
